### DISCUSSION TITLE
[help] Is it possible to specify targets to skip?

### DISCUSSION THREAD

**Discussion by daranzolin** [ropensci/targets#1039] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

I have a targets pipeline that culminates in several large reports. When I make changes to the pipeline, I want to rerun all the targets and inspect the output (with `tar_load`, `tar_read`, etc.) *except* the targets that run the reports. I currently am just commenting the targets out in `_targets` and running `tar_make` but that doesn't feel right. How do I say, 'run the first nine targets but not the 10th?'

**Comment by wlandau** [comment, marked as answer]
You could use cues: https://docs.ropensci.org/targets/reference/tar_cue.html. Or you could the `names` argument of `tar_make()` (https://docs.ropensci.org/targets/reference/tar_make.html#arguments) e.g. `tar_make(names = -any_of(c("report1", "report2")))`.

**Reply by daranzolin** [reply]
that's beautiful, thanks!
